### ISSUE TITLE
Editor controls appear to screen readers as plain text graphical icons

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7195]
### Where To Find Bug
Create (Editor)

### Expected Behaviour
Screen reader users should have a sense of how to use each icon/part of the editor.

### Actual Behaviour
Although these are labeled, screen reader users cannot be certain what icons have interactions.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Tab through the editor until you get to the editor controls
- Notice that the screen reader announces the name of the controls but not how to interact with them

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)

**Comment by benjiwheeler** [comment]
FWIW, I wasn’t able to understand this from the replication instructions. I tried replicating, but the description is vague as to what is expected vs what is experienced, and I’m not sure if I’m experiencing the problem.

**Comment by benjiwheeler** [comment]
It would help to have an example of a button that has this issue.
